integrationtests/tests/fixtures/mod.rs

```diff
@@ -216,18 +218,21 @@ pub async fn fixtures(num_peers: u16) -> anyhow::Result<Fixtures> {
 
             let user_db = if env::var("FM_CLIENT_SQLITE") == Ok(s) {
                 let db_name = format!("client-{}", rng().next_u64());
-                Database::new(sqlite(dir.clone(), db_name).await, module_decode_stubs())
+                Database::new(sqlite(dir.clone(), db_name).await, decoders.clone())
             } else {
-                Database::new(rocks(dir.clone()), module_decode_stubs())
+                Database::new(rocks(dir.clone()), decoders.clone())
```

Comment:
I should have not went on the mission to eliminate the `module_decode_stubs` usage here. I didn't understand the scope of work required. But I guess no point in reverting now.